### ISSUE TITLE
wheel on pypi requires 'futures' for python 3

### ISSUE THREAD

**Issue by danielpanteleit** [Azure/azure-storage-python#401]
Currently the wheel `azure_storage_blob-0.37.1-py2.py3-none-any.whl` requires `futures` even for python 3, because this expression from `setup.py`

```
(['futures'] if sys.version_info < (3, 0) else [])
```

is probably evaluated on python 2 when creating the wheel. There should be separate wheels for python 2 and python 3.

This also happens in `azure-storage-common`, by the way.

**Comment by zezha-msft** [comment]
Hi @danielpanteleit, thanks for bringing up the issue!

I'll take a look at update this thread.

**Comment by lmazuel** [comment]
@zezha-msft FYI, the recommended syntax now is:
```python
    extras_require={
        ":python_version<'3.4'": ['enum34>=1.0.4'],
    }
```
See https://github.com/Azure/msrest-for-python/blob/8a541abcb8bc889db91f7dc637b45dbb502f72a7/setup.py#L55-L57

**Comment by zezha-msft** [comment]
@lmazuel  Thanks!

**Comment by zezha-msft** [comment]
This was fixed in release 1.0.0.
